packages/components/src/Enumeration/Items/Item/Item.scss

```diff
@@ -55,7 +55,8 @@ $tc-enumeration-small-padding: $padding-small !default;
 	}
 
 	input {
-		margin: 10px;
+		line-height: 3.3rem;
```

Comment:
In the default state, a button wraps a item which define the height of a item with this property. So, I have added this property to get the same behavior.